### PR TITLE
e2e_framework:stop using deprecated wait.ErrwaitTimeout

### PR DESCRIPTION
**Pull request by wlq1212**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
e2e_framework:stop using deprecated wait.ErrwaitTimeout

#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes/kubernetes/pull/107826

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: test/e2e/common/node/node_lease.go
Comment (kerthcet) [inline test/e2e/common/node/node_lease.go:173]:
```suggestion
			if !wait.Interrupted(err) {
```

Enough I think?

Comment (wlq1212) [reply test/e2e/common/node/node_lease.go]: done

**Comment by SergeyKanzhelev** [comment]
/assign @tzneal 

for node files

**Review by neolit123 — COMMENTED** [review]
i see there were some problems with this new feature, but they were resolved:
https://github.com/kubernetes/kubernetes/pull/107826
https://github.com/kubernetes/kubernetes/issues/118671
https://github.com/kubernetes/kubernetes/pull/118686

/approve
/hold

please remove the hold once review has passed / lgtm.

**Review by tzneal — APPROVED** [review]
reviewed in detail

/lgtm

**Pull request merged**
